pkgs/top-level/python-packages.nix

```diff
@@ -1979,6 +1979,8 @@ self: super: with self; {
 
   convertdate = callPackage ../development/python-modules/convertdate { };
 
+  cookbook = callPackage ../servers/cookbook { };
```

Comment:
Is this a mere Python lib, or it also runs from command line?

Reply:
You could probably feasibly run it from the command line (in fact, that's the debug mode), but the more common way to run it is via some wsgi server (like uwsgi, see module), which then imports cookbook as a library.